src/reusable/CTooltip.re

```diff
@@ -150,7 +149,7 @@ module Styles = {
             top(`percent(50.)),
             transform(`translateY(`percent(-50.))),
             left(`px(-10)),
-            borderRightColor(Colors.gray7),
+            borderRightColor(`rgba((85, 85, 85, `num(1.0)))),
```

Comment:
@pzshine Should we use theme or use `rgba`?

Reply:
@evilpeach  P' @pzshine recommended to use hex instead ka since there's no similar color in theme